test/pi_image_overlay/etc/OpenVoiceOS/ovos.conf

```diff
@@ -0,0 +1,17 @@
+{
+    "module_overrides": {
+        "neon_core": {
+            "xdg": true,
```

Comment:
xdg is deprecated, always true now and cant be set to false, so this should be removed